webrtc/videoroom_advanced/lib/videoroom_web/views/layout_view.ex

```diff
@@ -1,3 +1,7 @@
 defmodule VideoRoomWeb.LayoutView do
   use VideoRoomWeb, :view
+
+  def version() do
+    Application.get_env(:membrane_videoroom_demo, :version)
```

Comment:
Just a nitpick, could be `Application.fetch_env!(:membrane_video_demo, :version)`